src/server/node.ts

```diff
@@ -128,7 +125,7 @@ export default class Node {
       this.info = nodeInformation.stats.info
 
       if (nodeInformation.stats.info.canUpdateHistory) {
-        this.info.canUpdateHistory = result(nodeInformation.stats, 'info.canUpdateHistory', false)
+        this.info.canUpdateHistory = nodeInformation.stats.info.canUpdateHistory || false
```

Comment:
this will error if stats is undefined

Reply:
yes, but it will occure in line 127 because it is accessed there as well, and never happend there. But let me do something to harden it.